sunshine/src/test/java/io/github/tatools/sunshine/SunshineConfigTest.java

```diff
@@ -0,0 +1,12 @@
+package io.github.tatools.sunshine;
+
+/**
+ * @author Dmytro Serdiuk (dmytro.serdiuk@gmail.com)
+ * @since 21.04.2017
+ */
+public class SunshineConfigTest {
+    @org.junit.Test
+    public void property() {
+        new SunshineConfig().property("dad");
```

Comment:
The goal is to verify, that SunshineConfig uses existent file to load data. In case the file is absent, we will see the Exception.